tests/mysql_test.go

```diff
@@ -194,35 +194,19 @@ func TestFetchBinlogFiles(t *testing.T) {
 		a.Equal(binlogFilesOnServerSorted[j].Size, binlogFilesDownloaded[j].Size)
 	}
 
-	t.Log("Truncate or delete downloaded files and re-download")
+	t.Log("Delete some downloaded files and re-download")
 	rand.Seed(time.Now().Unix())
-	// Fetch and randomly truncate/delete some binlog files.t.Log("Clean up binlog dir")
-	binlogFiles, err := os.ReadDir(binlogDir)
-	a.NoError(err)
-	for _, file := range binlogFiles {
-		path := filepath.Join(binlogDir, file.Name())
-		err = os.Remove(path)
-		a.NoError(err)
-	}
-	t.Log("Fetch binlog files")
-	err = mysqlDriver.FetchAllBinlogFiles(ctx, true /* downloadLatestBinlogFile */)
-	a.NoError(err)
-	binlogFilesDownloaded, err = mysqlDriver.GetSortedLocalBinlogFiles()
-	a.NoError(err)
-	t.Logf("Downloaded %d files to empty dir", len(binlogFilesDownloaded))
-	truncateIndex := rand.Intn(numBinlogFiles)
```

Comment:
Do not test for truncating files.

Reply:
Any AIs? Is this necessary to be in the same PR?

Reply:
We used to check binlog file size in `downloadBinlogFilesOnServer`.
Now we do not persist the file size in the metadata file, we do not check for the truncated file either.

Reply:
Should this be done in a separate PR, either before or after?

Reply:
Let's minimize the PR and have it focused focused.

Reply:
https://github.com/bytebase/bytebase/pull/2395